app/code/Magento/Store/Setup/InstallSchema.php

```diff
@@ -11,6 +11,7 @@
 use Magento\Framework\Setup\InstallSchemaInterface;
```

Comment:
`InstallSchema` is obsolete. Please update with latest mainline and use declarative schema

Reply:
Done.

Reply:
Fixed.